core/src/main/scala/pureconfig/package.scala

```diff
@@ -19,6 +19,9 @@ import pureconfig.error._
 
 package object pureconfig {
 
+  // an alias for a `Derivation` of a `ConfigReader`
+  private type DerivedConfigReader[A] = Derivation[ConfigReader[A]]
```

Comment:
I guess I was lazy there :) without the alias I'd have to replace the syntatic sugar of writing `def loadConfig[Config: DerivedConfigReader]` with `def loadConfig[Config](implicit reader: Derivation[ConfigReader[Config]])`. While I prefer the shorter syntax I agree that it makes the code more obscure to read for users. I'll change it.